src/test/java/com/box/sdk/MetadataTest.java

```diff
@@ -194,6 +194,78 @@ public void testMetadataDate(String dateString, Long dateLong) {
         }
     }
 
+    @Test
+    @Category(IntegrationTest.class)
+    public void testMetadataPrecisionFloat() {
+        final float expectedValueFloat = 1234567890f;
+        BoxAPIConnection api = new BoxAPIConnection(TestConfig.getAccessToken());
+
+        long timestamp = Calendar.getInstance().getTimeInMillis();
+        String templateKey = "precision" + timestamp;
+        String fieldKey = "testPrecision";
+
+        List<MetadataTemplate.Field> fields = new ArrayList<MetadataTemplate.Field>();
+        MetadataTemplate.Field valueField = new MetadataTemplate.Field();
+        valueField.setKey(fieldKey);
+        valueField.setType("float");
+        valueField.setDisplayName("Value Field");
+        fields.add(valueField);
+
+        MetadataTemplate template = MetadataTemplate.createMetadataTemplate(api, "enterprise",
+                templateKey, "Precision " + timestamp, false, fields);
+
+        Assert.assertEquals("float", template.getFields().get(0).getType());
+
+        // Add template to item
+        Metadata mdValues = new Metadata();
+        mdValues.add("/" + fieldKey, expectedValueFloat);
+        BoxFolder.Info folder = BoxFolder.getRootFolder(api).createFolder("Metadata Precision Test " + timestamp);
+        Metadata actualMD = folder.getResource().createMetadata(templateKey, mdValues);
+
+        Assert.assertEquals(templateKey, actualMD.getTemplateName());
+
+        final double actualValueDouble = actualMD.getFloat("/" + fieldKey);
```

Comment:
Should we deprecate the [getFloat()](https://github.com/box/box-java-sdk/blob/SDK-1404-use-double-for-metadata-numbers/src/main/java/com/box/sdk/Metadata.java#L347) and add a getDouble() method?